src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -278,10 +289,21 @@ consistent "has a" relationship is tracked by this ticket: https://broadworkbenc
     actions: Set[ResourceAction],
     roles: Set[ResourceRoleName],
     descendantPermissions: Option[Set[AccessPolicyDescendantPermissions]] = Option(Set.empty),
-    memberPolicies: Option[Set[PolicyIdentifiers]] = Option(Set.empty)
+    memberPolicies: Option[Set[PolicyInfoResponseBody]] = Option(Set.empty)
 ) {
   def getDescendantPermissions: Set[AccessPolicyDescendantPermissions] = descendantPermissions.getOrElse(Set.empty)
 }
+
+@Lenses final case class AccessPolicyMembershipRequest(
```

Comment:
Did we want to start the namespacing of requests and responses now, or in a follow-on ticket.

Reply:
I am happy to start with these now, but since we did not decide to version this change, I was not sure how we wanted to do it.

Reply:
@tlangs and I met on Gather and we decided to make a sam.model.api package and to move the affected request/response objects (in this PR) in there for the time being until we can circle back and do namespacing for our SamModel objects